src/plugin/modules/widgets/similarGenomes.js

```diff
@@ -34,17 +35,20 @@ function (
                 thead(
                     tr([
                         th('Distance'),
-                        th('Scientific name'),
-                        th('Database name'),
-                        th('Database ID')
+                        th('Name')
                     ])
                 ),
                 tbody(distances.map((each) => {
+                    // Check if each genome has a KBase ID; if so, construct a dataview link
```

Comment:
Not all NCBI Refseq genome IDs that are returned from the search results have a mapping back to KBase, so for now we can just give the scientific name + refseq ID with no link anywhere. We could potentially link off to NCBI, but I chose to hold off on that idea.